File: CS/Models/UploadControlValidationSettings.cs
using DevExpress.Web;

public class MyUploadControlValidationSettings {
    public static UploadControlValidationSettings Settings = new UploadControlValidationSettings() {
        AllowedFileExtensions = new string[] { ".jpg", ".jpeg", ".png", ".bmp", ".pdf", ".xml", ".txt", ".rtf", ".docx", ".xls", ".xlsx" },
        MaxFileSize = 4194304
    };
}

File: VB/Models/UploadControlValidationSettings.vb
Imports DevExpress.Web

Public Class MyUploadControlValidationSettings
	Public Shared Settings As New UploadControlValidationSettings() With {.AllowedFileExtensions = New String() { ".jpg", ".jpeg", ".png", ".bmp", ".pdf", ".xml", ".txt", ".rtf", ".docx", ".xls", ".xlsx" }, .MaxFileSize = 4194304}
End Class